teammapper-frontend/mmp/src/map/handlers/nodes.ts

```diff
@@ -806,6 +810,7 @@ export default class Nodes {
 
             if (graphic === false) {
```

Comment:
we could refactor this to `if (graphic)` as long as undefined isn't a special case which is different to false.

Reply:
This might be unnecessary considering https://github.com/b310-digital/teammapper/pull/369, where we remove `graphic` entirely.